### ISSUE TITLE
Window close button truncated on some windows

### ISSUE THREAD

**Issue by schorschii** [linuxmint/cinnamon#10940]
```
 * Cinnamon version 5.2.7
 * Distribution Mint 20.3
 * Graphics hardware *and* driver used: NVIDIA proprietary driver
 * 64 bit
 ```

**Issue**
On some windows with small title bars the close button is not displayed properly (truncated/cropped). Example: the Handbrake queue window.
![Bildschirmfoto vom 2022-07-17 19-29-39](https://user-images.githubusercontent.com/17480795/179417986-bd367d1e-0e9e-4ff6-be34-e19514ef98b3.png)

**Expected behaviour**
The close button should be entirely visible. For those windows with smaller title bars, it should be smaller to fit into the title bar.

**Comment by JosephMcc** [comment]
Just tested this under Cinnamon 5.4 and can't reproduce the issue.

**Comment by schorschii** [comment]
Yes, thats true. The current Cinnamon version 5.2.7 on Linux Mint 20.3 shows this issue and is reproducible in the current live system ISO. But when installing Cinnamon 5.4.4 via the PPA the issue is gone. So I'm closing it in hope that Cinnamon 5.4.4 will be released for Mint 20.3.

**Comment by JosephMcc** [comment]
The new version of Cinnamon will not be released to 20.x. It will be part of Mint21.
